.github/workflows/create-release.yml

```diff
@@ -35,3 +37,197 @@ jobs:
       - run: |
           echo "Saw Liquibase version ${{ steps.collect-data.outputs.liquibaseVersion }}"
           echo "Saw Extension version ${{ steps.collect-data.outputs.liquibaseVersion }}"
+
+  build:
+    name: "Build and Test"
+    runs-on: ubuntu-latest
+    needs: setup
+    outputs:
+      releaseSha: ${{ steps.get-release-sha.outputs.releaseSha }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          persist-credentials: false # otherwise, the token used is the GITHUB_TOKEN, instead of your personal token
+          fetch-depth: 0 # otherwise, you will fail to push refs to dest repo
+
+      - name: Cache Local Maven Repository
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+
+      - name: Cache Built Code
+        uses: actions/cache@v2
+        with:
+          key: built-code-${{ github.run_id }}
+          path: ./**/target
+
+      - name: Set up JDK
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8'
+          distribution: 'adopt'
+
+      - name: Configure git user
+        run: |
+          git config user.name "liquibot"
+          git config user.email "liquibot@liquibase.org"
+
+      - name: Download and install liquibase.jar
+        uses: dsaltares/fetch-gh-release-asset@master
+        with:
+          repo: "liquibase/liquibase"
+          version: "tags/v${{ needs.setup.outputs.liquibaseVersion }}"
+          file: "liquibase-${{ needs.setup.outputs.liquibaseVersion }}.jar"
+          target: "liquibase.jar"
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Install liquibase.jar
+        run: mvn -B org.apache.maven.plugins:maven-install-plugin:3.0.0-M1:install-file -Dfile=liquibase.jar
+
+      - name: Update pom.xml with release versions and commit changes
+        run: |
+          mvn -B versions:set -DnewVersion=${{ needs.setup.outputs.extensionVersion }} -DallowSnapshots=false -DoldVersion="*"
+          mvn -B versions:use-dep-version -Dincludes=org.liquibase:liquibase-core -DdepVersion=${{ needs.setup.outputs.liquibaseVersion }} -DforceVersion=true
+
+          git add pom.xml
+          git commit -m "Version Bumped to ${{ needs.setup.outputs.extensionVersion }}"
+          git tag -a -m "Version Bumped to ${{ needs.setup.outputs.extensionVersion }}" liquibase-postgresql-${{ needs.setup.outputs.extensionVersion }}
+          git push "https://liquibot:$GITHUB_TOKEN@github.com/$GITHUB_REPOSITORY.git" HEAD:${{ github.ref }} --follow-tags --tags
```

Comment:
This has the potential to be restricted by github. Depending on the trigger, github will sometimes restrict which keys are available to runners. It might be okay since it's not trigger by dependabot. https://github.blog/changelog/2021-02-19-github-actions-workflows-triggered-by-dependabot-prs-will-run-with-read-only-permissions/

Reply:
Yeah, will maybe need to see when/if that ends up hurting us. From the ways I've tried to run it so far it seems OK.